satellite/peer.go

```diff
@@ -352,6 +363,49 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, config *Config) (*
 		peer.Accounting.Rollup = rollup.New(peer.Log.Named("rollup"), peer.DB.Accounting(), config.Rollup.Interval)
 	}
 
+	{ // setup mail
+		// TODO(yar): test multiple satellites using same OAUTH credentials
+		mailConig := config.Mail
+
+		from, err := mail.ParseAddress(mailConig.From)
```

Comment:
handle the parsing of address inside either in auth or service, then it would avoid the `net/mail` import here